DOCS-2512 Tag Enforcement in Synthetics Settings (#11717)
DataDog/documentation: content/en/synthetics/settings/_index.md

If you have access to the [custom role feature][9], add your user to a custom role that includes `synthetics_default_settings_read` and `synthetics_default_settings_write` permissions for default settings. 

### Tag enforcement

<div class="alert alert-warning">
Tag enforcement is an advanced feature included in the Enterprise plan. For all other plans, contact your account representative or <a href="mailto:success@datadoghq.com">success@datadoghq.com</a> to request this feature.
</div>

> This allows you to enforce selected tags to create a Synthetics test. So you will be able to break down cost and usage by services, applications or teams for example. 

Reviewer: ```suggestion
This feature allows you to enforce selected tags to create a Synthetics test so you are able to break down cost and usage by services, applications, or teams, for example. 
```

You can activate tag enforcement in Synthetic Monitoring > Settings > Default Settings:

> <!-- Can you please embed this image here https://a.cl.ly/RBuLAdPY -->

Reviewer: <!-- Can you please embed this image here https://a.cl.ly/RBuLAdPY -->

Putting this as a comment here so it's more noticeable @alai97 😄
